### PR TITLE
Feat/alert button

### PR DESCRIPTION
**Pull request by Aditi0505**
### Alert
image

### Alert with close button
image

### Solid Button
image

### Outline Button
image

### Ghost Button
image

### Disabled Button
image

### Link Button
image

### Icon Button
image

### Floating Action Button
image

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+}
+.fa-times-circle {
+  color: var(--primary-color);
+}
+.btn-close {
+  background-color: transparent;
+  cursor: pointer;
+  border: 0;
+  padding: 0;
+}
```
Comment (anupkgurung) [inline components/alert/alert.css:50]: isn't 0 the default value of padding ?

**Review by Creator-singh — APPROVED** [review]
great work 🚀 good to go

**Review by anupkgurung — APPROVED** [review]
looks good !!

File: components/alert/alert.css (hunk 1 above)
Comment (Aditi0505) [reply components/alert/alert.css:50]: The button's default padding value isn't 0.

**Comment by Arpit-dotcom** [comment]
LGTM

**Pull request merged**